scss/titles.scss

```diff
@@ -0,0 +1,13 @@
+.title {
+  font-family: 'Roboto', sans-serif;
```

Comment:
We will use this font-family in every string in the project. So you should add 
@import url('https://fonts.googleapis.com/css?family=Roboto');
 * {
  font-family: 'Roboto', sans-serif;
} in `styles.scss`